dotnet/src/Skills/Skills.UnitTests/Planning/SequentialPlanner/PlanParserTests.cs

```diff
@@ -132,6 +132,7 @@ public void CanCallToPlanFromXml()
 </plan>";
 
         // Act
+        // TODO: unit tests should not depend on the code under test
```

Comment:
Same as other, not sure what the concern here is.

Reply:
see other comment, easier to discuss offline using IDE